src/cartogram_info/read_csv.cpp

```diff
@@ -307,15 +307,15 @@ void process_area_strs(
   }
 }
 
-void CartogramInfo::read_csv(const argparse::ArgumentParser &arguments)
+void CartogramInfo::read_csv(const Arguments &args)
 {
-  csv::CSVReader reader = load_csv(arguments);
+  csv::CSVReader reader = load_csv(args);
 
-  const int id_col = extract_id_header_col_index(reader, arguments);
-  const int area_col = extract_area_col_index(reader, arguments);
-  const int inset_col = extract_inset_col_index(reader, arguments);
-  const int color_col = extract_color_col_index(reader, arguments);
-  const int label_col = extract_label_col_index(reader, arguments);
+  const int id_col = extract_id_header_col_index(reader, args);
+  const int area_col = extract_area_col_index(reader, args);
+  const int inset_col = extract_inset_col_index(reader, args);
+  const int color_col = extract_color_col_index(reader, args);
+  const int label_col = extract_label_col_index(reader, args);
```

Comment:
As per my file comment, all these signatures should then look different, as they'll all have `args.MEMBER` as their final arguments.